Bottom: 9fdb90cffae23a13fdd35e0f449d9505a7d76974
Top:    e0fde573f00a9cfb0639c81146f8dd9c6a6eaa4b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-27 20:29:37 +0100

Comment out check for GUI HID when considering hidenames

Yes - I want to hide them for a print!


---

diff --git a/src/draw.c b/src/draw.c
index 32036d7..4f24674 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -188,7 +188,7 @@ DrawStrippedText (ElementTypePtr Element, int min_width)
 static void
 draw_element_name (ElementType *element)
 {
-  if ((TEST_FLAG (HIDENAMESFLAG, PCB) && gui->gui) ||
+  if ((TEST_FLAG (HIDENAMESFLAG, PCB) /*&& gui->gui*/) ||
       TEST_FLAG (HIDENAMEFLAG, element))
     return;
   if (doing_pinout || doing_assy)
